src/main/java/com/pingcap/tikv/PDClient.java

```diff
@@ -170,6 +165,7 @@ public Store getStore(long storeId) {
 
   @Override
   public void close() throws InterruptedException {
+    connPool.values().parallelStream().forEach(ManagedChannel::shutdown);
```

Comment:
For case above (mostly less than hundreds), parallelStream will not give you any bonus. I prefer to keep it simple and clean (without multithreading). Also other cases similar. Please considering how a language features implemented and then decide if to use.
Also in test, it seems managed channel cannot shutdown gracefully, so I removed this on purpose but added idle timer to close conn. So maybe we can remove this.